client/state/data-layer/utils.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import {
+	isArray,
+	mergeWith,
+} from 'lodash';
+
+const concatHandlers = ( left, right ) =>
+	isArray( left )
```

Comment:
can you explain this one? why does it return `undefined` when `left` is not an array?